core/tests/ledger_cleanup.rs

```diff
@@ -26,9 +26,9 @@ mod tests {
     };
 
     const DEFAULT_BENCHMARK_SLOTS: u64 = 50;
-    const DEFAULT_BATCH_SIZE: u64 = 1;
+    const DEFAULT_BATCH_SIZE_SLOTS: u64 = 1;
     const DEFAULT_MAX_LEDGER_SHREDS: u64 = 50;
-    const DEFAULT_ENTRIES_PER_SLOT: u64 = 500;
+    const DEFAULT_SHREDS_PER_SLOT: u64 = 25;
```

Comment:
Reading the print from running the tests before, the 500 entries resulted in 23 slots so this keeps the scale of the test similar